www/include/home/customViews/action.php

```diff
@@ -51,6 +51,28 @@
 
 $centreon = $_SESSION['centreon'];
 $action = $_POST['action'];
+
+$postFilter = array(
+    'widget_id' => array(
+        'filter' => FILTER_VALIDATE_INT,
+        'options' => array('default' => false)
```

Comment:
filter_validate_int will return false if it doesn't match an integer.
why adding the options ?